tensorflow/python/ops/math_grad.py

```diff
@@ -842,17 +842,19 @@ def _CrossGrad(op, grad):
 @ops.RegisterGradient("Cumsum")
 def _CumsumGrad(op, grad):
   axis = op.inputs[1]
+  exclusive = op.get_attr("exclusive")
   reverse = op.get_attr("reverse")
-  return [math_ops.cumsum(grad, axis=axis, reverse=(not reverse)), None]
+  return [math_ops.cumsum(grad, axis, exclusive, (not reverse)), None]
```

Comment:
Use keyword arg syntax for `exclusive` and `reverse`, and remove unnecessary parentheses around `not reverse`.